registry.k8s.io/images/k8s-staging-cluster-api/images.yaml

```diff
@@ -93,6 +93,7 @@
     "sha256:d93407d031296336ccbabc8494005672dc048c4ebc616ccfc18f813d49bd87fc": ["v1.10.5"]
     "sha256:46254acdd8a372b938c80a73fbc87c4473a4d6c142a97ab76728eae5d493171a": ["v1.10.6"]
     "sha256:3cda2bb5f363f60eb5a6b6d84d047c2afd1cce547e6b4fea445d590fee39779d": ["v1.10.7"]
+    "sha256:39e4db2cb287b67f953abf549557b7ca75234ba999f996c1f4d3bc4e75e164f2": ["v1.10.8"]
```

Comment:
SHA `39e4db2cb287` matches cluster-api-controller v1.10.8 at the [staging repo](https://console.cloud.google.com/gcr/images/k8s-staging-cluster-api/global/cluster-api-controller@sha256:39e4db2cb287b67f953abf549557b7ca75234ba999f996c1f4d3bc4e75e164f2/details).